docs/sources/writing-guide/documentation-structure/index.md

```diff
@@ -12,21 +12,74 @@ keywords:
 
-Information architecture (IA) refers to the way in which content is organized. Before you start contributing to product documentation, it is important to understand the IA of the content.
+Before you begin contributing to documentation, it is important to understand the structure of the content.
 
-Generally, the IA determines how content is:
+## User goals and documentation structure
+
+Any documentation you create should be focused on what your user's goals are. Refer to [Focus on user goals]({{< relref "../../style-guide/style-conventions" >}}) for more information.
+
+Your documentation structure should reflect these user goals. Think about what your users want to do, what they need to know, and how they can accomplish the tasks.
+
+This approach applies not only to content on a page but also to how you organize a set of documentation, whether it's for a product or a feature.
+
+With well-structured content, you can find what you need quickly and easily. Topics flow in a logical progression.
+
+<!-- Commenting this paragraph out until we have an actual section on information architecture.
+
+According to STC’s [Information Design and Information Architecture: Why Technical Communicators Should Care About These Fields](https://www.stc.org/intercom/2022/05/information-design-and-information-architecture-why-technical-communicators-should-care-about-these-fields/),
+information architecture is "...the practice of organizing, structuring, and labeling information to it’s easy to find, use, and understand..."
+-->
+
+## Structure of published content
+
+Generally, documentation structure determines how content is:
 
 - Titled
 - Grouped
 - Combined (or not combined) with other, related content
 
-The following examples are based on the Grafana OSS and Enterprise documentation.
+A standardized content structure that spans sets of documentation provides a consistent user experience.
+Information flows from a higher (less specific) level to a lower (more specific) level.
+For example, a new Grafana user wants to learn conceptual information first, so _Introduction_ comes before _Installation_.
+
+### Use the topics you need
+
+Depending on your product design and maturity, you may not need every topic listed below.
+If a topic does not apply to your project, you don't need to use it.
+For Grafana OSS for example, you might use all of the headings.
+For Grafana Enterprise Traces for example, you might only use a subset of topics.
+
+Some topics are optional and are usually found in specific contexts.
+For example, the _Create_, _Manage_, and _Monitor_ topics are used in Grafana OnCall but are not used in Grafana Tempo.
+
+### Topic list
+
+This table provides a list of the high-level topics used for grouping content. When writing new content, consider where it should appear given this content structure. For example, a conceptual page explaining metrics would go under the Introduction topic.
```

Comment:
```suggestion
You can use the following high-level topics to group content. When writing new content, consider where it should appear given this content structure. For example, a conceptual page explaining metrics would go under the Introduction topic.
```
Rationale: I lean towards talking less about the mechanics, such as "page", "table", "article", and more about the content.